tests/tensor/test_math.py

```diff
@@ -747,11 +747,12 @@ def test_isnan():
 
 class TestMaxAndArgmax:
     def setup_method(self):
-        MaxAndArgmax.debug = 0
+        Argmax.debug = 0
+        Max.debug = 0
```

Comment:
What do these do?

Reply:
Hm, it looks like a piece of old code.  I think I should to remove it.